local-volume/provisioner/pkg/common/common.go

```diff
@@ -102,3 +110,49 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 		},
 	}
 }
+
+// GetVolumeConfigFromConfigMap gets volume configuration from given configmap,
+func GetVolumeConfigFromConfigMap(client *kubernetes.Clientset, namespace, name string) (map[string]MountConfig, error) {
+	configMap, err := client.CoreV1().ConfigMaps(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return ConfigMapDataToVolumeConfig(configMap.Data)
+}
+
+// GetDefaultVolumeConfig returns the default volume configuration.
+func GetDefaultVolumeConfig() map[string]MountConfig {
+	return map[string]MountConfig{
+		"local-storage": {
+			HostDir:  DefaultHostDir,
+			MountDir: DefaultMountDir,
+		},
+	}
+}
+
+// VolumeConfigToConfigMapData converts volume config to configmap data.
+func VolumeConfigToConfigMapData(config map[string]MountConfig) (map[string]string, error) {
+	configMapData := make(map[string]string)
+	for class, data := range config {
```

Comment:
Initially I thought the same, but configmap.Data is a map[string][string]. which means config must be:
```
kind: ConfigMap
metadata:
  name: local-volume-config
  namespace: kube-system
data:
  "local-fast": |
    {
      "hostDir": "/mnt/ssds",
      "mountDir": "/local-ssds"
    }
  "local-slow": |
    {
      "hostDir": "/mnt/hdds",
      "mountDir": "/local-hdds"
    }
```

To parse with one call, we need following configmap, but it won't parse to configmap.Data
```
kind: ConfigMap
metadata:
  name: local-volume-config
  namespace: kube-system
data:
  '{"local-fast":
    {
      "hostDir": "/mnt/ssds",
      "mountDir": "/local-ssds"
    }
  "local-slow":
    {
      "hostDir": "/mnt/hdds",
      "mountDir": "/local-hdds"
    }
   }'
```

I didn't look closely into this, so correct me if i'm wrong